models/quality_measures/intermediate/nqf0053_osteoporosis_management_in_women_who_had_a_fracture/quality_measures__int_nqf0053_numerator.sql

```diff
@@ -0,0 +1,270 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False))))
+    | as_bool
+   )
+}}
+
+with denominator as (
+
+    select 
+          patient_id
+        , performance_period_begin
+        , performance_period_end
+        , recorded_date
+        , measure_id
+        , measure_name
+        , measure_version
+    from {{ref('quality_measures__int_nqf0053_denominator')}}
+
+)
+
+, value_sets as (
+
+    select
+          code
+        , code_system
+        , concept_name
+    from {{ ref('quality_measures__value_sets') }}
+    
+)
+
+, osteo_procedure_codes as (
+
+    select
+          code
+        , code_system
+        , concept_name
+    from value_sets
+    where lower(concept_name) in (
+          'bone mineral density test'
+        , 'bone mineral density tests cpt'
+        , 'bone mineral density tests icd10pcs'
+        , 'bone mineral density tests hcpcs'
+        , 'dexa dual energy xray absorptiometry, bone density'
+        , 'central dual energy x-ray absorptiometry (dxa)'
+        , 'spinal densitometry x-ray' 
+        , 'ultrasonography for densitometry' 
+        , 'ct bone density axial'
+        , 'peripheral dual-energy x-ray absorptiometry (dxa)'
```

Comment:
Can you add the concept `'osteoporosis medication'` to this list? There is a one hcpcs code in this value set related to pharma therapy for osteo. The concept name is medication, but it's related to a procedure code. 

> HCPCS code G8633 for Pharmacologic therapy (other than minerals/vitamins) for osteoporosis prescribed as maintained by CMS falls under Additional Quality Measures.

Reply:
Concept name `'osteoporosis medication'` has been added in the procedure codes filter.